### PR TITLE
Chore - move build variables to variable template

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to https://github.com/arcus-azure/arcus/issues/46

### CODE REVIEW

**Comment by stijnmoreels** [comment]
The build was not failed bc of this PR: https://dev.azure.com/codit/Arcus/_build/results?buildId=42188&view=logs&j=6f530bc7-0339-5c23-f0b1-5c642f3efaa5&t=dd031c7c-0a68-5383-196c-bc5d73ab51cf
Again something to do with updated Azure resources? @tomkerkhove ?

**Comment by stijnmoreels** [comment]
Waiting till https://github.com/arcus-azure/arcus.messaging/pull/79, logging could help pinpoint the issue.

File: src/Arcus.BackgroundJobs.CloudEvents/Arcus.BackgroundJobs.CloudEvents.csproj [hunk 1]
```diff
@@ -17,8 +17,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Arcus.EventGrid" Version="3.0.0-preview-1" />
-    <PackageReference Include="Arcus.Messaging.Pumps.ServiceBus" Version="0.1.0-preview-2" />
-    <PackageReference Include="Microsoft.Azure.EventGrid" Version="3.2.0" />
+    <PackageReference Include="Arcus.EventGrid" Version="3.0.0" />
+    <PackageReference Include="Arcus.Messaging.Pumps.ServiceBus" Version="20200312.0.0-PR-81" />
```
Comment (tomkerkhove) [inline src/Arcus.BackgroundJobs.CloudEvents/Arcus.BackgroundJobs.CloudEvents.csproj]: I'd suggest to ship a new preview version instead

File: src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj [hunk 2]
```diff
@@ -27,4 +27,9 @@
   <ItemGroup>
     <ProjectReference Include="..\Arcus.BackgroundJobs.CloudEvents\Arcus.BackgroundJobs.CloudEvents.csproj" />
   </ItemGroup>
+
+  <PropertyGroup>	
+    <RestoreSources>$(RestoreSources);https://api.nuget.org/v3/index.json;https://www.myget.org/F/arcus/api/v3/index.json</RestoreSources>
```
Comment (tomkerkhove) [inline src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj]: No need to add this, no? But can't hurt thought.

File: src/Arcus.BackgroundJobs.CloudEvents/Arcus.BackgroundJobs.CloudEvents.csproj (hunk 1 above)
Comment (stijnmoreels) [reply src/Arcus.BackgroundJobs.CloudEvents/Arcus.BackgroundJobs.CloudEvents.csproj]: Yeah, we could do that. Then we can remove the MyGet sources from the project files too.

**Comment by stijnmoreels** [comment]
Released, waiting to be indexed by NuGet before I push...

**Pull request merged**